lesson14/js/main.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+//Modal
+const menuLists = document.createElement('ul');
+
+function createModalWindow() {
+	const modalWindow = document.createElement('div');
+	const createMenuContainer = document.createElement('div');
+
+	//modal container
+	modalWindow.classList.add('bl_modal');
+	modalWindow.setAttribute('id', 'js-modalContainer');
+
+	//menu button
+	const createMenuBtn = document.createElement('button');
+	createMenuBtn.setAttribute('id', 'js-loadmenuBtn');
+	createMenuBtn.setAttribute('class', 'btn');
+	createMenuBtn.textContent = `MENU`;
+
+	//close button
+	const createCloseBtn = document.createElement('button');
+	createCloseBtn.setAttribute('id', 'js-closeModal');
+	createCloseBtn.setAttribute('class', 'el_closeModalBtn');
+	createCloseBtn.textContent = '×';
+
+	//input area
+
+	const createInputButton = document.createElement('input');
+	createInputButton.type = 'number';
+	createInputButton.placeholder = 'please input number';
+	createInputButton.min = '1';
+	createInputButton.setAttribute('id', 'number');
+	createInputButton.setAttribute('class', 'el_input');
+
+	createMenuContainer.setAttribute('id', 'js-menu_container');
+	const modal = document.createDocumentFragment();
+
+	modal.appendChild(createInputButton);
+	modal.appendChild(createMenuBtn);
+	modal.appendChild(createCloseBtn);
+	modal.appendChild(createMenuContainer);
+	modalWindow.appendChild(modal);
+	document.body.appendChild(modalWindow);
+
+	loadMenu();
+	closeModalWindow();
+}
+
+//open Modal
+const openModalBtn = document.getElementById('js-openModalBtn');
+
+openModalBtn.addEventListener(
+	'click',
+	() => {
+		const modalBg = document.createElement('div');
+		modalBg.id = 'js-bgFixed';
+		modalBg.classList.add('el_modalBg');
+		document.body.appendChild(modalBg);
+		createModalWindow();
+		document.body.style.position = 'fixed';
+	},
+	false
+);
+
+function loadMenu() {
+	const loadMenuBtn = document.getElementById('js-loadmenuBtn');
+	loadMenuBtn.addEventListener(
+		'click',
+		(event) => {
+			const inputNumber = document.getElementById('number');
+			if (!inputNumber.value === false) {
+				getInputValue();
+				callMenuContents();
+			} else {
+				alert('未入力です。');
+			}
+		},
+		false
+	);
+}
+
+function closeModalWindow() {
+	const closeModlBtn = document.getElementById('js-closeModal');
+
+	closeModlBtn.addEventListener(
+		'click',
+		() => {
+			const modalContainer = document.getElementById('js-modalContainer');
+			const modalWindow = document.createElement('div');
+			const bgFixed = document.getElementById('js-bgFixed');
+			removeInnerHTML(modalWindow);
+			removeInnerHTML(menuLists);
+			document.body.removeAttribute('style');
+			bgFixed.remove();
+			modalContainer.remove();
+		},
+		false
+	);
+}
+
+function getInputValue() {
+	const inputNumber = document.getElementById('number');
+	console.log(inputNumber.value);
+}
```

Comment:
IMO
It's not a get because it's just outputting and not returning numbers

Reply:
Thank you for your review.
I'm sorry that our response was delayed.

I changed the name here.
I understand that one function is up to the point where get returns a value in JavaScript.

[d0ef3b8](https://github.com/nyattsu72/hanson-frontend-cource/pull/19/commits/d0ef3b8b66c81795f48dc905d54137cfa2201a32)